docs/architecture/adr/ADR032-nlb-for-mtls.md

```diff
@@ -0,0 +1,19 @@
+# ADR032: NLB for mTLS
+
+## Status
+
+Accepted
+
+## Context
+
+Verify's [doc-checking](https://github.com/alphagov/doc-checking) serviceis secured in part using mTLS. Currently, our clusters are fronted by ALBs which cannot provide mTLS.
+
+The doc-checking service currently runs an nginx that provides the mTLS functionality from within the cluster. In order for GSP to be able to allow something within the cluster to perform mTLS we must run a load balancer that forwards unaltered TCP packets in addition to, or instead of an ALB.
```

Comment:
```suggestion
The doc-checking service currently runs an nginx that provides the mTLS functionality. In order for GSP to be able to allow something within the cluster to perform mTLS we must run a load balancer that forwards unaltered TCP packets in addition to, or instead of an ALB.
```